apps/aecore/src/aec_accounts_trees.erl

```diff
@@ -69,6 +74,11 @@ lookup(Pubkey, Tree) ->
 enter(Account, Tree) ->
     aeu_mtrees:enter(key(Account), value(Account), Tree).
 
+-dialyzer({nowarn_function, delete/2}).
```

Comment:
Maybe you want to include delete function - with spec - inside a `-ifdef(TEST).` too so to avoid warning on unused function?